recipes/dwave-neal/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "dwave-neal" %}
+{% set version = "0.5.9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/dwave-neal-{{ version }}.tar.gz
+  sha256: 6eabf37a7ccbed7f6145567351673a07f9ba5482a2500872c840b27089e0ff3e
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - setuptools >=46.4.0
+    - wheel
+  host:
+    - cython >=0.29.21,<3.0
+    - numpy >=1.19.1,<2.0.0,!=1.21.0,!=1.21.1
+    - pip
+    - python
+  run:
+    - dimod >=0.9.11
```

I see we have a bug in our [setup.py](https://github.com/dwavesystems/dwave-neal/blob/a11e477c3a6b3585d75ee8b58be75a4127d0c17c/setup.py#L67). We should use:
```suggestion
    - dimod >=0.9.11,<0.11.0
```